sim/druid/balance/TestBalance.results

```diff
@@ -46,911 +46,911 @@ character_stats_results: {
  value: {
-  dps: 7165.52089
-  tps: 7003.40788
+  dps: 7132.18218
+  tps: 6970.06918
  }
 }
 dps_results: {
  key: "TestBalance-AllItems-TearsoftheVanquished-47215"
  value: {
-  dps: 7168.72932
-  tps: 7011.22553
+  dps: 7133.82809
+  tps: 6976.32429
  }
 }
 dps_results: {
  key: "TestBalance-AllItems-TheGeneral'sHeart-45507"
  value: {
-  dps: 7122.2576
-  tps: 6961.16781
+  dps: 7088.9189
+  tps: 6927.82911
  }
 }
 dps_results: {
  key: "TestBalance-AllItems-ThunderheartHarness"
  value: {
-  dps: 3306.49091
-  tps: 3144.13968
+  dps: 3281.55077
+  tps: 3119.19955
  }
 }
 dps_results: {
  key: "TestBalance-AllItems-ThunderheartRegalia"
  value: {
-  dps: 5025.57247
-  tps: 4869.56177
+  dps: 4998.37426
+  tps: 4842.36356
  }
 }
 dps_results: {
  key: "TestBalance-AllItems-ThunderingSkyflareDiamond"
  value: {
-  dps: 7233.41235
-  tps: 7071.2084
+  dps: 7199.43499
+  tps: 7037.23104
  }
 }
 dps_results: {
  key: "TestBalance-AllItems-TinyAbominationinaJar-50351"
  value: {
-  dps: 7122.2576
-  tps: 6961.16781
+  dps: 7088.9189
+  tps: 6927.82911
  }
 }
 dps_results: {
  key: "TestBalance-AllItems-TinyAbominationinaJar-50706"
  value: {
-  dps: 7122.2576
-  tps: 6961.16781
+  dps: 7088.9189
+  tps: 6927.82911
  }
 }
 dps_results: {
  key: "TestBalance-AllItems-TirelessSkyflareDiamond"
  value: {
-  dps: 7276.08904
-  tps: 7112.84405
+  dps: 7242.11168
+  tps: 7078.8667
  }
 }
 dps_results: {
  key: "TestBalance-AllItems-TirelessStarflareDiamond"
  value: {
-  dps: 7267.5537
-  tps: 7104.51692
+  dps: 7233.57635
+  tps: 7070.53957
  }
 }
 dps_results: {
  key: "TestBalance-AllItems-TomeofArcanePhenomena-36972"
  value: {
-  dps: 7247.79719
-  tps: 7083.9476
+  dps: 7215.40103
+  tps: 7051.55145
  }
 }
 dps_results: {
  key: "TestBalance-AllItems-TrenchantEarthshatterDiamond"
  value: {
-  dps: 7267.5537
-  tps: 7104.51692
+  dps: 7233.57635
+  tps: 7070.53957
  }
 }
 dps_results: {
  key: "TestBalance-AllItems-TrenchantEarthsiegeDiamond"
  value: {
-  dps: 7276.08904
-  tps: 7112.84405
+  dps: 7242.11168
+  tps: 7078.8667
  }
 }
 dps_results: {
  key: "TestBalance-AllItems-UndeadSlayer'sBlessedArmor"
  value: {
-  dps: 5021.74043
-  tps: 4858.21629
+  dps: 4992.83294
+  tps: 4829.3088
  }
 }
 dps_results: {
  key: "TestBalance-AllItems-Val'anyr,HammerofAncientKings-46017"
  value: {
-  dps: 7741.02632
-  tps: 7576.7845
+  dps: 7704.78099
+  tps: 7540.53917
  }
 }
 dps_results: {
  key: "TestBalance-AllItems-VengefulGladiator'sIdolofResolve-33947"
  value: {
-  dps: 7355.53047
-  tps: 7194.44067
+  dps: 7320.79478
+  tps: 7159.70498
  }
 }
 dps_results: {
  key: "TestBalance-AllItems-WingedTalisman-37844"
  value: {
-  dps: 7272.43848
-  tps: 7096.9407
+  dps: 7239.09978
+  tps: 7063.60199
  }
 }
 dps_results: {
  key: "TestBalance-AllItems-WrathfulGladiator'sIdolofResolve-51429"
  value: {
-  dps: 7355.53047
-  tps: 7194.44067
+  dps: 7320.79478
+  tps: 7159.70498
  }
 }
 dps_results: {
  key: "TestBalance-Average-Default"
```

Comment:
Looks like balance is losing 34 DPS in the average test. Is this expected?

Reply:
If you look at the Remaining Differences commit, balance damage doesn't change at all there, so it should be fully explained by the caster trinket change.